Gruntfile.js

```diff
@@ -421,6 +444,144 @@ module.exports = function (grunt) {
 				}
 			}
 		},
+		shell: {
```

Comment:
move shell options into separate module. Gruntfile.js is becoming excessively large.